docs/configuration_file.md

```diff
@@ -65,6 +68,28 @@ Where:
 
 DaLI comes with a few builtin plugins, but more are needed: help us make DaLI a robust open-source, community-driven crypto data loader by [contributing](../CONTRIBUTING.md#contributing-to-the-repository) plugins for exchanges and wallets!
 
+### Binance.com Section (REST)
+This plugin is REST-based and requires setting up API Keys in your Binance.com account settings (click on the API link).
+
+**IMPORTANT NOTE**:
+* When setting up API key/secret, only use read permissions (DaLI does NOT need write permissions).
+* store your API key and secret safely and NEVER share it with anyone!
+
+Initialize this plugin section as follows:
+<pre>
+[dali.plugin.input.rest.binance_com <em>&lt;qualifiers&gt;</em>]
+account_holder = <em>&lt;account_holder&gt;</em>
+api_key = <em>&lt;api_key&gt;</em>
+api_secret = <em>&lt;api_secret&gt;</em>
+username = <em>&lt;username&gt;</em>
+native_fiat = <em>&lt;native_fiat&gt;</em>
+</pre>
+
+Notes: 
+* The `username` parameter is optional and denotes the username used when connecting to the Binance.com mining pool. Only basic mining deposits (type 2) are currently supported.
+* Due to a bug in the Binance.com system, there may be duplicate locked savings/staked and flexible savings dividends around May 8th, 2021. Please review your payments around this time before processing your transactions with RP2.
```

Comment:
Is there a bug tracker link on their site or somewhere else? It may be helpful to add it here.

Reply:
This is just a bug I've discovered on my own. Let me try to find if it is documented anywhere.

Reply:
Sounds good, but don't spend a lot of time searching for it. If you don't have an official link to the bug, we can move on.

Reply:
I asked them and they said the discrepancy is based on the old system timestamp is when the interest is technically delivered, but the new system reports when the interest affects the user's wallet. I added the link in the docs.